ontology/ncit.md

```diff
@@ -31,6 +31,7 @@ products:
 activity_status: active
 repository: https://github.com/NCI-Thesaurus/thesaurus-obo-edition
 preferredPrefix: NCIT
+domain: upper
```

Comment:
@mellybelly you're marked as the responsible person for this "ontology". What domain is really appropriate, since it covers basically everything?

Reply:
Noo! Put it in `health` or whatever that one was called. Not `upper`

Reply:
```suggestion
domain: health
```